terraform-cost-estimation

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+FILE_PATH=$1
+
+if [ $# != 1 ]
```

Comment:
Use `(( .. ))` for numerical comparisons (https://google.github.io/styleguide/shellguide.html#quoting).

```
if (( $# != 1 )) ; then
```